.github/workflows/docker.yml

```diff
@@ -69,10 +69,10 @@ jobs:
             dockerfile: synapse
             tags: "matrixdotorg/sytest-synapse:focal"
             build_args: "SYTEST_IMAGE_TAG=focal"
-          - sytest_image_tag: buster
+          - sytest_image_tag: bullseye
             dockerfile: synapse
-            tags: "matrixdotorg/sytest-synapse:buster"
-            build_args: "SYTEST_IMAGE_TAG=buster"
+            tags: "matrixdotorg/sytest-synapse:bullseye"
+            build_args: "SYTEST_IMAGE_TAG=bullseye"
```

Comment:
I'm somewhat surprised that we don't need `
SYSTEM_PIP_INSTALL_SUFFIX=--break-system-packages
` here, but if it works then LGTM.

Reply:
My assumption is that it works? The dockerfiles seemed to build but I could be missing something here?

Reply:
IIRC I added the `SYSTEM_PIP_INSTALL_SUFFIX` stuff to get sytest working under debian testing. I had assumed bullseye was brand-new latest release and so would need this too, but that's not the case: [bullseye is nearly two years old](https://endoflife.date/debian).

So yeah, given that the docker images build this is fine. Sorry for the noise.

Reply:
Not at all I appreciate the insight!